src/mongocrypt-ctx-encrypt.c

```diff
@@ -1947,18 +1948,22 @@ _try_empty_schema_for_create (mongocrypt_ctx_t *ctx)
  * }
  * If the "create" command does not include a JSON schema, an empty JSON schema
  * is returned. This is to avoid an unnecessary 'listCollections' command for
- * create. */
+ * create.
+ *
+ * If the "collMod" command does not include a JSON schema, a schema is later
+ * requested from other sources. This is because a "collMod" command may have
```

Comment:
Can "from other sources" be made more specific, e.g. mention the `MONGOCRYPT_CTX_NEED_MONGO_COLLINFO` state?

Reply:
Done.